client/src/Movies/MovieCard.js

```diff
@@ -1,7 +1,24 @@
 import React from 'react';
 
 const MovieCard = props => {
-  return;
+  return (
+    <div className="movie-card">
+        <h2>{props.movie.title}</h2>
+        <div className="movie-director">
+          Director: <em>{props.movie.director}</em>
+        </div>
+        <div className="movie-metascore">
+          Metascore: <strong>{props.movie.metascore}</strong>
+        </div>
+        <h3>Actors</h3>
+        {props.movie.stars.map(star => (
+          <div key={star} className="movie-star">
+            {star}
+          </div>
+        ))}
+        {/* <button onClick={props.saveMovie}>Save Movie</button> */}
```

Comment:
I'd love to see you get this functionality implemented! It's a stretch, I know, but it will be a great way to flex your javaScript muscles and get some problem solving in.